src/app/templates/app/submit_new_license.html

```diff
@@ -70,7 +70,7 @@
 
     <div class = "form-group">
       <div class="col-sm-12">
-      <label class="control-label col-sm-3" >Briefly explain how this license <br />meets the
+      <label class="control-label col-sm-3" >Briefly explain how this license or exception<br />meets the
```

Comment:
Isn't it better to have either shown license or exception?
Like if the dropdown selected is license then we show only `Briefly explain how this license` and when selected exception - `Briefly explain how this exception`. 
@jlovejoy Let me know your thoughts. I think since the dropdown option comes later in the form, it might get confusing.

@BassCoder2808 Please do the needful if needed. Overall LGTM. Will merge after @jlovejoy approvals.